app/lib/widgets/polls/comment_admin.dart

```diff
@@ -0,0 +1,192 @@
+import 'package:flutter/material.dart';
+import 'package:opengov_app/service/http_service.dart';
+import 'package:opengov_app/widgets/base/dialogs.dart';
+import 'package:opengov_app/widgets/base/linked_text.dart';
+import 'package:opengov_app/widgets/base/list_header.dart';
+import 'package:opengov_app/widgets/polls/edit_poll.dart';
+import 'package:opengov_app/widgets/polls/neapolitan.dart';
+import 'package:opengov_common/actions/delete_poll.dart';
+import 'package:opengov_common/actions/update_comment.dart';
+import 'package:opengov_common/actions/update_reply.dart';
+import 'package:opengov_common/models/comment.dart';
+import 'package:opengov_common/models/poll.dart';
+import 'package:opengov_common/models/reply.dart';
+
+class CommentAdmin extends StatefulWidget {
+  final Comment comment;
+
+  const CommentAdmin({required this.comment});
+
+  @override
+  _CommentAdminState createState() => _CommentAdminState();
+}
+
+class _CommentAdminState extends State<CommentAdmin> {
+  late Comment _comment;
+  Iterable<Reply>? _moderationQueue;
+  Iterable<Reply>? _approvedReplies;
+
+  @override
+  void initState() {
+    super.initState();
+    _comment = widget.comment;
```

Comment:
Is there a reason why you store the comment in the state rather than just referring to `widget.comment`? I usually do the latter, but maybe this is better practice.